packages/es-components/src/components/controls/checkbox/Checkbox.js

```diff
@@ -86,24 +88,47 @@ const CheckboxText = styled.span`
   margin-left: 5px;
 `;
 
+const AdditionalHelpContent = styled.div`
+  font-size: ${props => props.theme.sizes.baseFontSize};
+  font-weight: 400;
+  margin: 10px 0 10px 0;
+  text-transform: none;
+  flex-basis: 100%;
+`;
+
 function Checkbox({
+  name,
   labelText,
   value,
   isChecked,
   isDisabled,
   onClick,
   onChange,
   ariaLabel,
+  validationState,
+  additionalHelpContent,
```

Comment:
Since we're explicitly defining the normal input props rather than allowing props to pass through to the input element, we may want to include others here as well such as `id` and `required`.

The alternative would be to remove all the typical input properties (including `name`, `onChange`, `onClick`, `checked`, etc.) and describe in the documentation that it allows them. I sort of prefer this because we're not doing anything special with any of the props, just passing them through anyway, so this method would clean up a lot of the code. You'd be left with:

```
<CheckboxInput type="checkbox" focusBorderColor={theme.colors.inputFocus} {...checkboxProps} />
```

Reply:
Should we also remove `isDisabled` in favor of folks passing down `disabled`?  Same question for `isChecked` and `checked`.

Reply:
Since I am using `name` for the additional help content id, it will fail prop type validation if I don't explicitly define that particular one.

Reply:
I was thinking we could remove isDisabled and isChecked, yeah.